dist/index.html

```diff
@@ -22,8 +22,8 @@
         <!-- Add your site or application content here -->
         <p>Hello world! This is HTML5 Boilerplate.</p>
 
-        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
-        <script>window.jQuery || document.write('<script src="js/vendor/jquery-1.11.2.min.js"><\/script>')</script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
```

Comment:
> this is still 404'ing

@arthurvr Yes I know, that's why I haven't merge this.

--

> The team sent mails to the google CDN guys so generally this will be fixed quick.

Cc: @paulirish